### PR TITLE
Javascript javascript1 week3

### PR DESCRIPTION
**Pull request by Divyajg**

### CODE REVIEW

File: javascript/javascript1/week3/travel.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+function travel() {
```
Comment (TITHI1244) [inline javascript/javascript1/week3/travel.js:6]: Try to name the functions more descriptive. Think about what it is doing here, and start with a verb, maybe calculateTravelTime.

File: javascript/javascript1/week3/travel.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    let result1 = parseInt(travelInformation.destinationDistance / travelInformation.speed);
+    let result2 = (travelInformation.destinationDistance % travelInformation.speed);
```
Comment (TITHI1244) [inline javascript/javascript1/week3/travel.js:8]: You are not asked to use the variable `travelInformation` inside the function directly. You were supposed to use it as the function parameter. Try to pass it as the function argument.

File: javascript/javascript1/week3/travel.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    return result1 + " hours and " + result2 + " minutes";
+}
+
+const travelTime = travel(travelInformation);
```
Comment (TITHI1244) [inline javascript/javascript1/week3/travel.js:12]: JavaScript is very flexible, so it's not showing you any error. But your function `travel` does not accept any parameter and here you are passing one argument. Try to fix it :)

File: javascript/javascript1/week3/series.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+];
+const avgLifespan = 80;
+const leapyearsInLifespan = parseInt(avgLifespan / 4);
+const normalyearsInLifespan = (avgLifespan - leapyearsInLifespan);
+const daysInLifespan = (leapyearsInLifespan * 366 + normalyearsInLifespan * 365);
+const hoursInLifespan = daysInLifespan * 24;
+const minInLifespan = exactLifespan = hoursInLifespan * 60;
+
+
+function percentage() {
```
Comment (TITHI1244) [inline javascript/javascript1/week3/series.js:28]: Again, a little more descriptive function name :)

File: javascript/javascript1/week3/series.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        hours: 21,
+        minutes: 9,
+    },
+];
+const avgLifespan = 80;
+const leapyearsInLifespan = parseInt(avgLifespan / 4);
+const normalyearsInLifespan = (avgLifespan - leapyearsInLifespan);
+const daysInLifespan = (leapyearsInLifespan * 366 + normalyearsInLifespan * 365);
+const hoursInLifespan = daysInLifespan * 24;
+const minInLifespan = exactLifespan = hoursInLifespan * 60;
```
Comment (TITHI1244) [inline javascript/javascript1/week3/series.js:25]: Where does `exactLifespan` come from and why do you need that? If you need that then remove the first variable `minInLifespan`.

File: javascript/javascript1/week3/notes.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    }
+    return result;
+}
+
+
+console.log(getNote(3));
+console.log(getNote(5));
+console.log(getNote(" "));
+console.log(getNote("ght"));
+
```
Comment (TITHI1244) [inline javascript/javascript1/week3/notes.js:39]: Nice 👍

File: javascript/javascript1/week3/smartphone.js [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+addActivity("Youtube", 30);
+addActivity("Facebook", 20);
+addActivity("Google-news", 100);
+addActivity("Instagram", 60);
+
+console.log(activities);
+
+function showStatus() {
```
Comment (TITHI1244) [inline javascript/javascript1/week3/smartphone.js:19]: I think, you meant to do `function showStatus(activities)` here, right?

**Review by TITHI1244 — CHANGES_REQUESTED** [review]
Very good job. A few improvements needed. Try to implement those 👍

File: javascript/javascript1/week3/item-array-removal.js [hunk 8]
```diff
@@ -12,7 +12,7 @@ const nameToRemove = "Ahmad";
 
 // Write some code here
 for (let i = 0; i < names.length; i++) {
-    if (names[i] === "Ahmad") {
+    if (names[i] === nameToRemove) {
```
Comment (TITHI1244) [inline javascript/javascript1/week3/item-array-removal.js:15]: 👍

File: javascript/javascript1/week3/series.js [hunk 9]
```diff
@@ -20,28 +20,28 @@ const seriesDurations = [{
...
-const daysInLifespan = (leapyearsInLifespan * 366 + normalyearsInLifespan * 365);
-const hoursInLifespan = daysInLifespan * 24;
-const minInLifespan = exactLifespan = hoursInLifespan * 60;
+const lifespanInDays = (leapyearsInLifespan * 366 + normalyearsInLifespan * 365);
+const lifespanInHours = lifespanInDays * 24;
+const lifespanInMinutes = lifespanInHours * 60;
 
 
-function percentage() {
+function percentageOfTimeSpentOnSeries(seriesDurations) {
```
Comment (TITHI1244) [inline javascript/javascript1/week3/series.js:28]: I would name it `calculateTimeSpentOnSeries` starting with the verb calculate :)

File: javascript/javascript1/week3/smartphone.js [hunk 10]
```diff
@@ -16,7 +16,7 @@ addActivity("Instagram", 60);
 
 console.log(activities);
 
-function showStatus() {
+function showStatus(activities) {
```
Comment (TITHI1244) [inline javascript/javascript1/week3/smartphone.js:19]: 👍

File: javascript/javascript1/week3/travel.js [hunk 11]
```diff
@@ -3,11 +3,11 @@ const travelInformation = {
     destinationDistance: 432,
 };
 
-function travel() {
+function calculateTravelTime(travelInformation) {
```
Comment (TITHI1244) [inline javascript/javascript1/week3/travel.js:6]: 👍

**Review by TITHI1244 — APPROVED** [review]
Nice job 👍

File: javascript/javascript1/week3/series.js (hunk 9 above)
Comment (Divyajg) [reply javascript/javascript1/week3/series.js:28]: Thank you, I will remember these for next time.

**Pull request merged**